DTS-STN/Alpha-Site
# Anonymous feedback

# Description

[#472 Create anonymous feedback component](https://trello.com/c/X9ywObnp/472-472-create-anonymous-feedback-component)

I did the feedback component and linked it to the phase banner. It doesn't submit anything yet.
![image](https://user-images.githubusercontent.com/72703030/128925997-61604f66-225b-4155-a7d6-593732023c26.png)
![image](https://user-images.githubusercontent.com/72703030/128926092-a202fd33-168c-4a02-b41b-864d028014f4.png)
![image](https://user-images.githubusercontent.com/72703030/128926607-95567dd6-075d-407f-9fe7-9c1f673c3259.png)
![image](https://user-images.githubusercontent.com/72703030/128926681-a736bb01-11b3-4a55-b639-0cc25f1bd6d0.png)

## Acceptance Criteria

It should like the image that is in this task on trello (https://trello.com/c/G3pLR1BW/29-29-qualitative-product-owner-feedback)

## Test Instructions

1. There isn't any active experiments yet, so try it in storybook or in the layout component, go to the Feedback and change the active prop for true.
2. Check how it looks like on big and small screens compared to the picture here (https://trello.com/c/G3pLR1BW/29-29-qualitative-product-owner-feedback)

## Help Requested

- Things that you (the submitter)...
- want reviewers to pay very close...
- attention to when they review this PR.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
